src/test/java/com/tabnine/plugin/completionPostProcess/Spaces.kt

```diff
@@ -0,0 +1,51 @@
+package com.tabnine.plugin.completionPostProcess
+
+import com.tabnine.binary.requests.autocomplete.postprocess
+import org.junit.Test
+
+class Spaces {
```

Comment:
kapara give it a bit of a more meaningful name

Reply:
ah, the test's full name is combined with the package name - which gives it the meaning